src/Arcus.Messaging.Pumps.Abstractions/MessagePump.cs

```diff
@@ -168,6 +182,26 @@ protected void NotifyResumeRetrievingMessages()
             Logger.LogDebug("Circuit breaker caused message pump '{JobId}' to transition back from '{CurrentState}' to a 'Closed' state, retrieving messages is resumed", JobId, CircuitState);
 
             CircuitState = MessagePumpCircuitState.Closed;
+
+            NotifyCircuitBreakerEventHandlers();
+        }
+
+        private void NotifyCircuitBreakerEventHandlers()
```

Comment:
I'm nitpicking, but we actually **trigger** the eventhandlers and by doing so, we **notify** the subscribers. 
So, I think this method should be named `TriggerCircuitBreakerStateChangedEventHandlers` or `NotifyCircuitBreakerStateChangedSubscribers`

Reply:
Done that here: https://github.com/arcus-azure/arcus.messaging/pull/461/commits/5fef61ad7cefd3d09ed2c02cdf63cca4982313e5